frontend/src/Settings/Indexers/Indexers/EditIndexerModalContent.js

```diff
@@ -43,13 +43,14 @@ function EditIndexerModalContent(props) {
     enableInteractiveSearch,
     supportsRss,
     supportsSearch,
-    fields
+    fields,
+    priority
   } = item;
 
   return (
     <ModalContent onModalClose={onModalClose}>
       <ModalHeader>
-        {`${id ? 'Edit' : 'Add'} Indexer - ${implementationName}`}
+        {`${id ? translate('Edit') : translate('Add')} ${translate('Indexer')} - ${implementationName}`}
```

Comment:
Note: there are a few other cases like this in Settings that likely need translations (for later). Also, should we do it like this to make sure its accurate for all other languages (some languages orient phrases differently):

 `{`${id ? translate('EditIndexer') : translate('AddIndexer')} - ${implementationName}`}`